docs/config.md

```diff
@@ -1247,12 +1247,12 @@ The above configuration snippet sets 2 cpus for the processes annotated with the
 
 #### Selector priority
 
-When mixing generic process configuration and selectors the following priority rules are applied (from lower to higher):
+Process configuration settings are resolved as follows (from lowest to highest priority):
```

Comment:
Should we include also settings defined when including processes from modules?

See e.g. https://github.com/nextflow-io/nextflow/pull/4188